### DISCUSSION TITLE
A very stupid bash question: using a variable as an argument

### DISCUSSION THREAD

**Discussion by aborruso** [johnkerl/miller#660] [Q&A] [answered]
Hi all,
I have this input csv file

```
a a,b,c
1,2,a
3,5,t
4,7,o
```

If I run `mlr --csv cut -f "a a","b" input.csv`  I have

```
a a,b
1,2
3,5
4,7
```

If I run in the shell these commands I have instead an empty result.

```
p="\"a a\",\"b\""
mlr --csv cut -f "$p" input.csv
```

How do I must use a variable as an argument?

**Comment by NikosAlexandris** [comment, marked as answer]
Andrea, why not exclude the double quotes (as they are not appearing in your `input.csv`):
```bash
➜  p="a a,b" ;echo "$p"
a a,b

➜  mlr --csv cut -f "$p" input.csv
a a,b
1,2
3,5
4,7
```

**Reply by aborruso** [reply]
I feel very very stupid.

The single mlr command is `mlr --csv cut -f "a a","b" input.csv`.  If I run it without double quotes - `mlr --csv cut -f a a,b input.csv` - it does not work.

For this reason, I have added the escaped double quotes in the variable: `p="\"a a\",\"b\""`.

But I'm stupid, it's not necessary, bash does everything right and magically for me.

Thank you @NikosAlexandris
